wg-notebooks/charter.md

```diff
@@ -0,0 +1,39 @@
+# WG Notebooks Charter
```

Comment:
It might be better to write a README file, which indicates the Meetings / Organizers / Contacts etc.

[KF-Serving README](https://github.com/kubeflow/community/tree/daebbb8b2ca22f91f22bb2a759a0501b14aff1d8/wg-serving) should be a good example.

Reply:
@PatrickXYS WG's README is generated by [generator](https://github.com/kubeflow/community/tree/master/generator).

@kimwnasptd Just need to run `make generate` under `/generator` folder and it should create all appropriate files from updated `wgs.yaml`

Reply:
thanks for the suggestions and instructions @PatrickXYS @andreyvelich!